src/common/components/Tag/Tag.theme.ts

```diff
@@ -0,0 +1,66 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type TagVariants = VariantProps<typeof tagTheme>;
+
+export const tagTheme = tv(
+  {
+    slots: {
+      tag: [
+        "inline-flex",
+        "min-w-14",
+        "py-[0.125rem]",
+        "px-3",
+        "gap-2",
+        "justify-center",
+        "items-center",
+        "rounded-[6.1875rem]",
+        "border",
+        "border-solid",
+        "border-border-default",
+      ],
+      icon: [],
+    },
+    variants: {
+      variant: {
+        secondary: {
+          tag: [
+            "border-border-secondary",
+            "bg-border-secondary",
+            "text-text-on-secondary",
+          ],
+        },
+      },
+      active: {
+        true: {
+          tag: [
+            "border-border-primary",
+            "bg-border-primary",
+            "text-text-on-primary",
+          ],
+        },
+      },
+      hoverable: {
+        true: {
+          tag: ["hover:border-border-secondary", "hover:bg-border-secondary"],
+        },
+      },

```

Comment:
Figma design looks simpler where we only have 1 variant, that has no background on default, then `bg-element-secondary` on hover, and `bg-primary-default` on active.

Reply:
If we want to add more variants, I feel they should maybe have their own hover and active states instead of sharing them! Now it looks a bit difficult to trace what will happen to each variant (default or secondary) on hover and active

Reply:
alright! I've removed the `hoverable` state and `secondary` variant so now we'll only have default, hover and active states. updated at fcd81341